tests/buildkite/build-cpu-arm64.sh

```diff
@@ -18,7 +18,7 @@ $command_wrapper bash -c "cd build && ctest --extra-verbose"
 
 echo "--- Build binary wheel"
 $command_wrapper bash -c \
-  "cd python-package && rm -rf dist/* && python setup.py bdist_wheel --universal"
+  "cd python-package && rm -rf dist/* && pip wheel --no-deps -v . --wheel-dir dist/"
```

Comment:
Minor note, since you're using `pip wheel --no-deps` everywhere you may be able to use `python -m build -w` as a drop-in replacement for the most part.

Reply:
You may also be able to use that in your documentation and CI scripts, but I don't have a strong preference either way there.